apps/mac.c

```diff
@@ -93,6 +119,16 @@ int mac_main(int argc, char **argv)
             if (opts == NULL || !sk_OPENSSL_STRING_push(opts, opt_arg()))
                 goto opthelp;
             break;
+        case OPT_CIPHER:
+            cipher = alloc_mac_algorithm_name(&opts, "cipher", opt_arg());
```

Comment:
This looks like a slippery slope to me :)

Reply:
Wheeee!!!